charts/opentelemetry-demo/templates/_pod.tpl

```diff
@@ -46,6 +46,18 @@ Get Pod Env
 {{- define "otel-demo.pod.env" -}}
 {{- $prefix := include "otel-demo.name" $ }}
 
+{{- if eq .name "featureflag-service" }}
+{{- $hasDatabaseUrl := false }}
+{{- range .env }}
+{{- if eq .name "DATABASE_URL" }}
+{{- $hasDatabaseUrl = true }}
+{{- end}}
+{{- end }}
+{{- if not $hasDatabaseUrl }}
+{{- $_ := set . "env" (append .env (dict "name" "DATABASE_URL" "value" (printf "ecto://ffs:ffs@%s-ffs-postgres:5432/ffs" $prefix)) | uniq) }}
```

Comment:
I don't think `uniq` will do the trick that you expect. If user provides a custom `DATABASE_URL`, only `name` key/value pair is the same, but the `value` pair will likely be different, so the items won't be identical for `uniq` to be applied. If you provide two `env` items with the same `name`, Kubernetes doesn't provide any guarantee which one is applied and behaves pretty wonky based on my experience.

Reply:
Ya we probably don't even need it because of the logic above. The logic above ensures that we only ever modify the list if DATABASE_URL was never present so I think we can safely remove the call to uniq. What do you think?

Reply:
Also, I think we should refactor something to make setting env var easier. Adding env vars for specific services that rely on other services (and therefore the Release name) doesn't feel graceful.

Reply:
@dmitryax done.